packages/tradershub/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5TradeLink.tsx

```diff
@@ -0,0 +1,82 @@
+import { Button, Text } from '@deriv/quill-design';
+import { getPlatformFromUrl } from '../../../../../helpers/urls';
+import { THooks, TPlatforms } from '../../../../../types';
+import { AppToContentMapper, PlatformDetails, PlatformToLabelIconMapper, PlatformUrls } from '../../../constants';
+
+type TMT5TradeLinkProps = {
+    app?: keyof typeof AppToContentMapper;
+    platform?: TPlatforms.All;
+    webtraderUrl?: THooks.MT5AccountsList['webtrader_url'];
+};
+
+const MT5TradeLink: FC<TMT5TradeLinkProps> = ({ app = 'linux', platform, webtraderUrl = '' }) => {
+    const content = AppToContentMapper[app];
+    const { data: ctraderToken } = useCtraderServiceToken();
+
+    const { data: activeAccount } = useActiveTradingAccount();
+
+    const isDemo = activeAccount?.is_virtual;
+
+    const mt5Platform = PlatformDetails.mt5.platform;
+    const dxtradePlatform = PlatformDetails.dxtrade.platform;
+    const ctraderPlatform = PlatformDetails.ctrader.platform;
+
+    const onClickWebTerminal = () => {
+        const { isStaging, isTestLink } = getPlatformFromUrl();
+        let url;
+        switch (platform) {
+            case dxtradePlatform:
+                url = isDemo ? PlatformUrls.dxtrade.demo : PlatformUrls.dxtrade.live;
+                break;
+            case ctraderPlatform:
+                url = isTestLink || isStaging ? PlatformUrls.ctrader.staging : PlatformUrls.ctrader.live;
+                if (ctraderToken) url += `?token=${ctraderToken}`;
+                break;
+            default:
+                url = '';
+        }
+        window.open(url);
+    };
+
+    return (
+        <div className='flex items-center justify-between border-t border-system-light-secondary-background px-800 py-1200'>
+            <div className='flex items-center gap-800'>
+                {(platform === mt5Platform || app === ctraderPlatform) && (
+                    <React.Fragment>
+                        <div className='w-1600 h-1600'>{content.icon}</div>
+                        <Text size='sm'>{content.title}</Text>
+                    </React.Fragment>
+                )}
+                {platform !== mt5Platform && app !== ctraderPlatform && (
+                    <Text size='sm'>Run {PlatformDetails[platform ?? dxtradePlatform].title} on your browser</Text>
+                )}
+            </div>
+            {(platform === mt5Platform || app === ctraderPlatform) && (
+                <Button
+                    onClick={() => window.open(app === 'web' ? webtraderUrl : content.link)}
+                    size='sm'
+                    variant='outlined'
+                >
+                    {content.text}
+                </Button>
+            )}
+            {platform !== mt5Platform && app !== ctraderPlatform && (
+                <Button
+                    className='flex items-center border-none rounded-md cursor-pointer bg-system-dark-primary-background p-400 gap-400'
```

Comment:
is there a reason for `flex items-center` here

Reply:
referring to the styles from wallets:
<img width="310" alt="image" src="https://github.com/binary-com/deriv-app/assets/108507236/3dfafa60-2755-4bf3-b97a-e9baac52140b">